package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "@pod-point/cache-js",
-  "version": "1.0.0",
+  "version": "1.0.1-5",
```

Comment:
is this how we should version it?

Reply:
does not seem standard

Reply:
Agreed tbh, I was just following suit, so I think for now it makes sense to continue with that.

Since this PR merges into initial, and not master, I think makes sense to just follow versioning that was done up until that point. Then when initial gets merged into master I'd recommend bumping minor version.

Then again, I wouldn't mind at all to do 1.0.2 instead tbh - especially because this is quite different to last version 🤔 

Either way, I'm cool with whatever you think I should do.

Reply:
I think 1.0.2 makes sense

Reply:
Adjusted